bot/tests/conftest.py

```diff
@@ -50,6 +50,11 @@ def __init__(self, nb):
         def as_markdown(self):
             return "This is the mock issue n°{}".format(self.nb)
 
+        def as_error(self):
```

Comment:
You are re-implementing the logic from the coverity `as_error` here, you need to use directly a `CoverityIssue` to trigger the real code. 
Otherwise the underlying implementation may change and the unit test won't catch the change

To do that, avoid using `mock_issues`, but instanciate directly a list of `CoverityIssue`